app/assets/javascripts/templates.js

```diff
@@ -0,0 +1,4 @@
+window.templates = {};
+window.templates['admin/shared/tagging/_breadcrumb_list'] = new Hogan.Template(function(c,p,i){var _=this;_.b(i=i||"");if(_.s(_.f("breadcrumbs",c,p,1),c,p,0,16,113,"{{ }}")){_.rs(c,p,function(c,p,_){_.b("<div class=\"taxon-breadcrumb\">");_.b("\n" + i);_.b("  <ol>");_.b("\n" + i);if(_.s(_.d(".",c,p,1),c,p,0,65,91,"{{ }}")){_.rs(c,p,function(c,p,_){_.b("      <li>");_.b(_.v(_.d(".",c,p,0)));_.b("</li>");_.b("\n");});c.pop();}_.b("  </ol>");_.b("\n" + i);_.b("</div>");_.b("\n");});c.pop();}return _.fl();;});
```

Comment:
It doesn't feel like this should be here.  Perhaps it was leftover from some other task and added by mistake?